server/src/test/java/org/elasticsearch/indices/state/RareClusterStateIT.java

```diff
@@ -315,6 +309,7 @@ public void testDelayedMappingPropagationOnReplica() throws Exception {
             Settings.builder()
                 .put(DiscoverySettings.COMMIT_TIMEOUT_SETTING.getKey(), "30s") // explicitly set so it won't default to publish timeout
                 .put(DiscoverySettings.PUBLISH_TIMEOUT_SETTING.getKey(), "0s") // don't wait post commit as we are blocking things by design
+                .put(MappingUpdatedAction.INDICES_MAPPING_DYNAMIC_TIMEOUT_SETTING.getKey(), "2s")
```

Comment:
can you elaborate on the rational of how you got to 2s?

Reply:
it's high enough so that the cluster state update task will actually get to run. It's low enough so that the test still finishes in a reasonable amount of time.